app/templates/components/url-box.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="ui action input" style="width: 100%; height: 36px">
-  <input class="truncate" style="flex: 1 0 !important" type="text" value={{@url}} placeholder={{@url}} readonly>
+  <input class="truncate" style={{html-safe (concat "flex: 1 0 !important;" @extraStyles)}} type="text" value={{@url}} placeholder={{@url}} readonly>
```

Comment:
why

Reply:
```
<input class="truncate" style={{"flex: 1 0 !important;" @extraStyles)}} type="text" value={{@url}} placeholder={{@url}}>
```

Sir this ember thrown this error on runtime , when i was adding some more styles to the url-box.

```
Binding style attributes may introduce cross-site scripting vulnerabilities; please ensure that values being bound are properly escaped. 
```
Then i used concat to add styles

```
Concatenated styles must be marked as `htmlSafe`.  style-concatenation
```
that's why i used html safe keyword

if i will not use extra stylings then, outcome will be like this

![Screenshot_2021-01-14 js engines Events Open Event](https://user-images.githubusercontent.com/65965202/104535113-8e932800-563b-11eb-8ba5-5e843078aa5f.png)

but mario sir asked me to keep the positions and everything same.